ui/models/bugJobMap.js

```diff
@@ -11,16 +11,17 @@ export default class BugJobMapModel {
 
   // the options parameter is used to filter/limit the list of objects
   static getList(options) {
-    return fetch(`${uri}${createQueryParams(options)}`).then(resp =>
-      resp.json().then(data => data.map(elem => new BugJobMapModel(elem))),
+    return fetch(`${getProjectUrl(uri)}${createQueryParams(options)}`).then(
```

Comment:
Should this use the `getData` wrapper?

Reply:
These should be converted, yeah.  I'll do that in a follow-up PR for Bug 1473777.  I'll do that one next.